packages/ui/src/ui/store/actions/settings/settings.ts

```diff
@@ -142,6 +142,12 @@ export function setSettingsSystemRpcProxiesCollapsed(value: boolean): SettingThu
     };
 }
 
+export function setSettingsSystemCypressProxiesCollapsed(value: boolean): SettingThunkAction {
+    return (dispatch) => {
+        dispatch(setSetting(SettingName.SYSTEM.CYPRESS_PROXIES_COLLAPSED, NAMESPACES.SYSTEM, value));
```

Comment:
We prefer to avoid `setSetting` action in new code, it is a js-legacy, its type definition is not enough strong. Please replace it with `setSettingByKey`.